### PR TITLE
fix: Apply pre-line into the OpenChannelUserMessage

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[UIKIT-2544](https://sendbird.atlassian.net/browse/UIKIT-2544)

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [x] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)

### CODE REVIEW

File: src/smart-components/OpenChannel/context/hooks/useSendMessageCallback.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ function useSendMessageCallback(
     if (sdk) {
       const text = messageInputRef.current.innerText;
       const createParamsDefault = (txt: string | number): UserMessageCreateParams => {
-        const message = typeof txt === 'string' ? txt.trim() : txt.toString(10).trim();
+        const message = (txt as string)?.trim() || txt as string;
```
Comment (sravan-s) [inline src/smart-components/OpenChannel/context/hooks/useSendMessageCallback.ts:29]: no chance of number coming in?

File: src/ui/OpenchannelUserMessage/index.tsx [hunk 2]
```diff
@@ -196,7 +178,17 @@ export default function OpenchannelUserMessage({
             type={LabelTypography.BODY_1}
             color={LabelColors.ONBACKGROUND_1}
           >
-            {MemoizedMessageText()}
+            {message?.message}
```
Comment (sravan-s) [inline src/ui/OpenchannelUserMessage/index.tsx:181]: 👍

**Review by sravan-s — APPROVED** [review]

**Comment by sravan-s** [comment]
lets merge this to main first ~

**Comment by HoonBaek** [comment]
https://github.com/sendbird/sendbird-uikit-react/pull/362#pullrequestreview-1174948013

**Pull request merged**